netket/nn/activation.py

```diff
@@ -48,6 +48,21 @@
 from jax.numpy import cosh
 from jax.numpy import sinh
 
+from netket.jax import HashablePartial
+
+
+@HashablePartial
+def reim(f):
+    # Modifies a non-linearity to act seperately on the real and imaginary parts
```

Comment:
Make it a docstring.
Also add it to https://github.com/netket/netket/blob/master/docs/docs/api.rst so that it shows up in the documentation.

(BTW, also add reim_relu, reim_selu to the website)